.github/workflows/maven.yml

```diff
@@ -18,7 +18,7 @@ jobs:
         with:
           java-version: 1.8
       - name: Build with Maven
-        run: mvn -B package --file pom.xml
+        run: mvn -B install --file pom.xml
```

Comment:
The build failed because it contains changes used by the samples. The samples retrieve their dependencies via maven central and therefore they fail to build.
This change makes the build succeed because the dependencies are being installed locally.

@nenaraab Are the dependencies not installed in the build environment on purpose? If so we have to revert this commit and it will only work build again after a release.

Reply:
No, i think this should be ok and will help us a lot to keep our pipeline running, even after version update 👍